### ISSUE TITLE
Support `Content-Encoding` header (compression)

### ISSUE THREAD

**Issue by ethanaobrien** [terreng/simple-web-server#74]
Less data is transferred (send dependency will need to be removed first)

for gzip I have found [this](https://www.npmjs.com/package/node-gzip) works

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding

**Comment by terreng** [comment]
This sounds great! Just make sure to check the `Accept-Encoding` request header to only use encoding with supported browsers.

**Comment by terreng** [comment]
I've added the HTTP compression option to the UI and the documentation (and renamed the option to `compression`). I made a slight [change to handlers.js](https://github.com/terreng/simple-web-server/pull/65/commits/d6b3c0dd736154464e556ed25ed7a001c866a9fa).

Unfortunately, now that I am trying it out, I'm finding that it's unbearably slow. Even on my machine. Is there any way to speed this up? Otherwise I don't think we should ship this feature. Maybe once we are using Tauri and running a more native web server it'll perform better.
